case-studies/README.md

```diff
@@ -0,0 +1,24 @@
+# SIG-Security Case Studies
+
+## Objective
+
+SIG-Security work ought to be informed by data and feedback on what works and what doesn't.
+This will help create a feedback loop between the theoretical/hypothetical and practical/tactical.
+This falls into 2 categories: process examples (e.g. how well did an assessment go?) and security practices examples (irrespective of the assessment, how are tools working out in the wild?)
+
+## Vision
+* Document example case studies, both uses of where things went right, and where things went wrong
+* Define a process for protecting the guilty (for the wrong cases) and preventing spam (for vendors or FOSS projects just looking to market their wares indesciriminately)
```

Comment:
Nit: vision is not to define a process, the vision is for the process to result in awesomeness

Suggestion:
* The process of collecting case studies results in knowledge-sharing that reduces security risks, providing good data about vendors and open source projects

I do think "vendors or FOSS projects just looking to market their wares indiscriminately" is a good point, but was a bit conflicted about putting it that way... maybe we could break that out as an issue using "suggestion" or "proposal" template to better define the process we've been following